### PR TITLE
issue-formAction

### PR DESCRIPTION
**Pull request by ellietms**
please let me know if you think it is not correct, as this is my first PR 🎉
Thanks a lot.

### CODE REVIEW

File: src/button/Button.tsx [hunk 1]
```diff
@@ -56,6 +56,10 @@ type ButtonProps = React.HTMLAttributes<HTMLButtonElement> & {
    * allow to add an image after the label when loading
    */
   customLoadingPostImage?: JSX.Element;
+  /**
+   * allow to add an image after the label when loading
```
Comment (daniele-zurico) [inline src/button/Button.tsx:60]: please change the comment on top

File: src/button/__tests__/Button.test.tsx [hunk 2]
```diff
@@ -33,6 +33,7 @@ const DummyLoadingButton = ({ loadingLabel }: any) => {
       customPostfixImg={<img id="postfixmg" alt="" src="" />}
       customLoadingPostImage={<img id="postLoadingImg" alt="" src="" />}
       customLoadingPreImage={<img id="preLoadingImg" alt="" src="" />}
+      formAction={""}
```
Comment (daniele-zurico) [inline src/button/__tests__/Button.test.tsx:36]: not sure why you need this... you not using it

File: src/button/Button.tsx (hunk 1 above)
Comment (ellietms) [reply src/button/Button.tsx]: Done

File: src/button/__tests__/Button.test.tsx (hunk 2 above)
Comment (ellietms) [reply src/button/__tests__/Button.test.tsx]: Done

**Comment by Ibabalola** [comment]
Hi @ellietms - we will also need to update the corresponding `.mdx` stories for the `button` component to demostrate how this new property works, please have a look at the `stories/Button/` folder

**Comment by daniele-zurico** [comment]
> Hi @ellietms - we will also need to update the corresponding `.mdx` stories for the `button` component to demostrate how this new property works, please have a look at the `stories/Button/` folder

I suggest to change only the live version?

**Comment by ellietms** [comment]
> Hi @ellietms - we will also need to update the corresponding `.mdx` stories for the `button` component to demostrate how this new property works, please have a look at the `stories/Button/` folder

I have updated my code please review it, Thanks @Ibabalola @daniele-zurico

**Pull request merged**